### PR TITLE
Moonshae Pixie & Myrkul's Edict

### LINKED ISSUE #899: Moonshae Pixie // Pixie Dust

### LINKED ISSUE #902: Myrkul's Edict

### PR DESCRIPTION
**Pull request by squee1968**
-closes #899
-closes #902

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Myrkul's Edict
+ManaCost:1 B
+Types:Sorcery
+A:SP$ RollDice | Sides$ 20 | ResultSubAbilities$ 1-9:OneOppSac,10-19:EachOppSac,20:SacTopPower | SpellDescription$ Roll a d20.
+SVar:OneOppSac:ChoosePlayer | Defined$ You | Choices$ Player.Opponent | SubAbility$ DBSac | StackDescription$ SpellDescription | SpellDescription$ 1—9 VERT Choose an opponent. That player sacrifices a creature.
+SVar:DBSac:DB$ Sacrifice | Defined$ Chosen | SacValid$ Creature
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt:6]: this should then cleanup chosen player... see Bend or Break for an example

File: forge-gui/res/cardsfolder/upcoming/moonshae_pixie.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Name:Moonshae Pixie
+ManaCost:3 U
+Types:Creature Faerie
+PT:2/2
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters the battlefield, draw cards equal to the number of opponents who were dealt combat damage this turn.
+SVar:TrigDraw:AB$ Draw | NumCards$ X
+SVar:X:PlayerCountRegisteredOpponents$HasPropertywasDealtCombatDamageThisTurn
+Oracle:Flying\nWhen Moonshae Pixie enters the battlefield, draw cards equal to the number of opponents who were dealt combat damage this turn.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/moonshae_pixie.txt:9]: misses: `AlternateMode:Adventure`

File: forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+SVar:EachOppSac:DB$ Sacrifice | Defined$ Player.Opponent | SacValid$ Creature | StackDescription$ SpellDescription | SpellDescription$ 10—19 VERT Each opponent sacrifices a creature.
+SVar:SacTopPower:DB$ RepeatEach | RepeatPlayers$ Player.Opponent | RepeatSubAbility$ DBChooseCard | SubAbility$ DBSacAll
+SVar:DBChooseCard:DB$ ChooseCard | Defined$ Player.IsRemembered | Choices$ Creature.greatestPowerControlledByRemembered | ChoiceTitle$ Choose a creature you control with the greatest power | Mandatory$ True | RememberChosen$ True
+SVar:DBSacAll:DB$ SacrificeAll | ValidCards$ Card.IsRemembered | SubAbility$ DBCleanup | StackDescription$ SpellDescription | SpellDescription$ 20 VERT Each opponent sacrifices a creature with the greatest power among creatures that player controls.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt:10]: Is this looking right in the stack description and card detail with the SpellDescription here? Seems to me the SpellDescription should be in line 8

File: forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | ClearChosen$ True
+Oracle:Roll a d20.\n1—9 | Choose an opponent. That player sacrifices a creature.\n10—19 | Each opponent sacrifices a creature.\n20 | Each opponent sacrifices a creature with the greatest power among creatures that player controls.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt:12]: probably add `SVar:NeedsToPlay:Creature.OppCtrl`

File: forge-gui/res/cardsfolder/upcoming/moonshae_pixie.txt [hunk 5]
```diff
@@ -0,0 +1,19 @@
+Name:Moonshae Pixie
+ManaCost:3 U
+Types:Creature Faerie
+PT:2/2
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters the battlefield, draw cards equal to the number of opponents who were dealt combat damage this turn.
+SVar:TrigDraw:AB$ Draw | NumCards$ X
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/moonshae_pixie.txt:7]: @kevlahnota probably needs to be DB

File: forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Name:Myrkul's Edict
+ManaCost:1 B
+Types:Sorcery
+A:SP$ RollDice | Sides$ 20 | ResultSubAbilities$ 1-9:OneOppSac,10-19:EachOppSac,20:SacTopPower | SpellDescription$ Roll a d20.
+SVar:OneOppSac:ChoosePlayer | Defined$ You | Choices$ Player.Opponent | SubAbility$ DBSac | StackDescription$ SpellDescription | SpellDescription$ 1—9 VERT Choose an opponent. That player sacrifices a creature.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt:5]: @kevlahnota `DB$ ` is missing

**Pull request merged**